src/test/java/org/apache/commons/lang3/builder/DiffBuilderTest.java

```diff
@@ -32,6 +32,31 @@
     
     private static final ToStringStyle SHORT_STYLE = ToStringStyle.SHORT_PREFIX_STYLE;
 
+    private static class IdentifiableTest {
```

Comment:
"Test" isn't such a got suffix for a class used in testing. How about "ObjectId" or something like that?